SAMtags.tex

```diff
@@ -182,7 +183,7 @@ \subsection{Additional Template and Mapping data}
 
 \item[Q2:Z:\tagvalue{}]
```

Comment:
Thanks.  Although there's a bigger howler than I noticed anyway.  The E2/U2 go together and R2/Q2, not E2/Q2 as my PR states.  I'll fix both of these.